Makefile

```diff
@@ -34,16 +45,21 @@ lint: $(TOOL_BIN)/golangci-lint
 
 .PHONY: docker
 docker:
-	cd docker && docker buildx build --load --no-cache --platform linux/arm64 -t ghcr.io/viam-modules/raspberry-pi:arm64 .
+	cd docker && docker buildx build --load --no-cache --platform linux/$(DOCKER_ARCH) -t ghcr.io/viam-modules/raspberry-pi:$(DOCKER_ARCH) .
+
+docker-32-bit:
+	cd docker && docker build --platform linux/arm -t ghcr.io/viam-modules/raspberry-pi:arm32 .
```

Comment:
The one above is variable... don't duplicate it here. Just set DOCKER_ARCH in an alias if anything.

```
docker-32:
     DOCKER_ARCH=arm make docker
```
(may need tweaking in syntax, writing from memory)